sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/serializer/DigitalTwinsStringSerializer.java

```diff
@@ -1,56 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.digitaltwins.core.implementation.serializer;
-
-import com.fasterxml.jackson.core.JsonGenerator;
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.SerializerProvider;
-import com.fasterxml.jackson.databind.ser.std.StdSerializer;
-
-import java.io.IOException;
-
-/**
- * Configures the {@link StdSerializer} with custom behavior needed to work with Digital Twins REST API.
- */
-public final class DigitalTwinsStringSerializer extends StdSerializer<String> {
```

Comment:
It might be good to verify if any of the existing tests used this type (we can check the jacoco coverage report to see if this class had any coverage). If none of the tests used this type, then we need to find out how this was used and what happens with the migration to azure-json.